account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -113,6 +117,20 @@ class AccountBankStatementLine(models.Model):
     aggregate_id = fields.Integer(compute="_compute_reconcile_aggregate")
     aggregate_name = fields.Char(compute="_compute_reconcile_aggregate")
 
+    @api.depends("move_id")
+    def _compute_allowed_manual_model_ids(self):
+        reconcile_model = self.env["account.reconcile.model"]
+        base_domain = [("rule_type", "=", "writeoff_button")]
+        for item in self:
+            domain = base_domain
+            if item.move_id:
```

Comment:
Why going to `move_id` instead of using `journal_id`? And on that mood, the domain can be static, not with this compute.